[2025-02-14T22:55:51.209+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T22:55:51.223+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T22:55:09.834374+00:00 [queued]>
[2025-02-14T22:55:51.231+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T22:55:09.834374+00:00 [queued]>
[2025-02-14T22:55:51.232+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T22:55:51.242+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-14 22:55:09.834374+00:00
[2025-02-14T22:55:51.245+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=886) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T22:55:51.246+0000] {standard_task_runner.py:72} INFO - Started process 887 to run task
[2025-02-14T22:55:51.247+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-14T22:55:09.834374+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpticet2zx']
[2025-02-14T22:55:51.248+0000] {standard_task_runner.py:105} INFO - Job 343: Subtask assign_permissions_dag
[2025-02-14T22:55:51.281+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T22:55:09.834374+00:00 [running]> on host 71404c6b5320
[2025-02-14T22:55:51.360+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T22:55:09.834374+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T22:55:09.834374+00:00'
[2025-02-14T22:55:51.361+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T22:55:54.622+0000] {dag.py:216} INFO - Assigned permissions to teste for DAG hello_world_dag
[2025-02-14T22:55:57.852+0000] {dag.py:216} INFO - Assigned permissions to teste for DAG hello_world_dag
[2025-02-14T22:55:57.854+0000] {logging_mixin.py:190} INFO - current_owner formanage_***_roles = data-team
[2025-02-14T22:55:57.855+0000] {dag.py:198} INFO - Permission for DAG manage_***_roles is already correctly assigned to data-team
[2025-02-14T22:55:57.855+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T22:55:57.863+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T22:55:57.863+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-14T22:55:09.834374+00:00, execution_date=20250214T225509, start_date=20250214T225551, end_date=20250214T225557
[2025-02-14T22:55:57.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T22:55:57.917+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T22:55:57.920+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
